web/src/beta/lib/core/engines/Cesium/utils.ts

```diff
@@ -151,6 +189,63 @@ export function findEntity(
   return;
 }
 
+export function findFeaturesFromLayer<T = Entity | Cesium3DTileset | InternalCesium3DTileFeature>(
+  viewer: CesiumViewer,
+  layerId: string,
+  featureId: string[],
+  convert?: (e: Entity | Cesium3DTileset | InternalCesium3DTileFeature) => T,
+): NonNullable<T>[] | undefined {
```

Comment:
Instead of returning undefined, consider returning an empty array to keep the return 
Same as above.[OPTIONAL]